src/is_win.rs

```diff
@@ -22,9 +22,8 @@ mod test {
         assert_eq!(is_win(2, 1, &two), false);
         assert_eq!(is_win(3, 1, &two), true);
         assert_eq!(is_win(1, 1, &half), true);
-        assert_eq!(is_win(1, 3, &third), true);
-        assert_eq!(is_win(1000, 10, &tenth), false);
-        assert_eq!(is_win(1001, 10, &tenth), true);
-        assert_eq!(is_win(1000, 500, &one), false);
+        assert_eq!(is_win(1, 3, &thirdish), true);
```

Comment:
I still don't think this is right; 2 for vs 1 against is "tied" therefor fails because no votes * power = yes votes. But that applies to this line as well (3 no votes * 1/3 power = 1 yes vote), so it should return false. The other two lines below seem to be right.

Reply:
The solution may be to approximate 1/3rd as 0.333...3334